src/components/dialogs/FileCorrectionDialog.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <v-dialog v-model="dialog" width="45rem" persistent :attach="attach" content-class="file-correction-dialog">
+    <v-card>
+      <v-card-title class="pt-8">Correction Filing</v-card-title>
+
+      <v-card-text class="font-15">
+        <p>Select who caused the correction. If client, completing party information and certification
+          will be required. If staff, completing party information and certification will not be required.</p>
+        <v-radio-group v-model="correctionType">
+          <v-radio id="correct-client-radio" class="mb-0 pt-2" label="Client Error" :value="CorrectionTypes.CLIENT" />
+          <v-radio id="correct-staff-radio" class="mb-0 pt-2" label="Staff Error" :value="CorrectionTypes.STAFF" />
+        </v-radio-group>
+        <template v-if="!hasChosenCorrection">
+          <p class="font-15 option-error">Choose one option to proceed</p>
+        </template>
+      </v-card-text>
+
+      <v-card-actions class="py-10">
+        <v-btn outlined large
+          class="mr-2"
+          color="primary"
+          id="dialog-exit-button"
+          @click="exit()"
+        > Cancel </v-btn>
```

Comment:
You shouldn't have spaces around the text.

With no spaces, you can "expect the text toBe('Cancel')" (otherwise you have to test with the spaces, or else use "toContain()" which is not exact).

If you did this to get extra whitespace around the text, use padding (or min-width) instead.